### PR TITLE
#379 Remove last reference to Bradford Font

### LINKED ISSUE #379: Remove outdated and unused fonts
**Desired Behavior**

- remove woff to reduce footprint
- Remove book font as it is unused

### PR DESCRIPTION
**Pull request by fthobe**
## Summary

Removing last reference in tailwind config to outdated fonts. 

Fully Fixes #379 
Fixes visualisation error visible in following comment (serif classes not rendered correctly): [https://github.com/solidusio/solidus_starter_frontend/pull/380#issuecomment-2490358724](https://github.com/solidusio/solidus_starter_frontend/pull/380#issuecomment-2490358724)

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [X] I have written a thorough PR description.
- [X] I have kept my commits small and atomic.
- [X] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

**Review by jarednorman — APPROVED** [review]

**Comment by fthobe** [comment]
Can we merge this?

**Comment by jarednorman** [comment]
Requested core review so we can get a second review.

**Review by tvdeyen — APPROVED** [review]

**Pull request merged**
